R/scale-date.r

```diff
@@ -275,7 +275,16 @@ scale_y_time <- function(name = waiver(),
   )
 }
 
-## rename to datetime_scale
+#' Date/time scale constructor
+#'
+#' @inheritParams scale_x_datetime
+#' @inheritParams continuous_scale
+#' @param trans For date/time scales, the name of a date/time transformation or
+#'   the object itself. Built-in transformations include "hms", "date" and "time".
+#' @inheritDotParams continuous_scale -aesthetics -trans -palette -breaks -minor_breaks -labels -guide
+#'
+#' @export
+#' @keywords internal
```

Comment:
I might be a fool, but I don't understand the pkgdown error:

> ! All topics must be included in reference index
✖ Missing topics: datetime_scale
ℹ Either add to _pkgdown.yml or use `@keywords internal`

Whereas adding the keywords is exactly what I've done

Reply:
Is it possible you ran `document()` before adding `@keywords internal` to the roxygen comments? It doesn't look like it made it through to the .Rd file.

Reply:
Whoops, my bad: a fool it is then! Sharp eye, thanks for catching this!